x/tokenfactory/keeper/admins_test.go

```diff
@@ -72,3 +74,221 @@ func (suite *KeeperTestSuite) TestAdminMsgs() {
 	suite.Require().NoError(err)
 	suite.Require().Equal("", queryRes.AuthorityMetadata.Admin)
 }
+
+func (suite *KeeperTestSuite) TestMintDenom() {
+	suite.SetupTest()
+	var (
+		denom     string
+		msgServer types.MsgServer
+		addr0bal  = int64(0)
+	)
+
+	// Create a denom
+	msgServer = keeper.NewMsgServerImpl(*suite.App.TokenFactoryKeeper)
```

Comment:
1. We should probably define and setup `msgServer` in the `suite` itself upon `SetupTest`
2. Also, why are we making explicit calls to `SetupTest`? Testify will automatically call `SetupTest` prior to each test.

Reply:
Sure. I'm going to change it